modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1,1190 +1,1167 @@
 
-=== Parameters
 
-
-[%header%autowidth.spread]
+==== Parameters
+[%header,cols="20s,20a,30a,20a,10a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Table Name a| String |  The name of the table containing the requested item. |  |x
-| Key a| Object |  A map of attribute names to AttributeValue objects, representing the primary key of the item to retrieve. |  |x
-| Attributes To Gets a| Array of String |  This is a legacy parameter. Use ProjectionExpression instead. |  |
-| Consistent Read a| Boolean |  Determines the read consistency model: If set to true, then the operation uses strongly consistent reads; otherwise, the operation uses eventually consistent reads. |  false |
-| Expression Attribute Names a| Object |  One or more substitution tokens for attribute names in an expression. The following are some use cases for using ExpressionAttributeNames: |  |
-| Projection Expression a| String |  A string that identifies one or more attributes to retrieve from the table. |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Table Name a| String |  The name of the table containing the requested item. |  | x
+| Key a| Object |  A map of attribute names to AttributeValue objects, representing the primary key of the item to retrieve. |  `#[payload]` | 
+| Attributes To Gets a| Array of String |  This is a legacy parameter. Use ProjectionExpression instead. |  | 
+| Consistent Read a| Boolean |  Determines the read consistency model: If set to true, then the operation uses strongly consistent reads; otherwise, the operation uses eventually consistent reads. |  false | 
+| Expression Attribute Names a| Object |  One or more substitution tokens for attribute names in an expression. The following are some use cases for using ExpressionAttributeNames: |  | 
+| Projection Expression a| String |  A string that identifies one or more attributes to retrieve from the table. |  | 
 | Return Consumed Capacity a| Enumeration, one of:
 
 ** INDEXES
 ** TOTAL
-** NONE |  Determines the level of detail about provisioned throughput consumption that is returned in the response. |  |
-| Target Variable a| String |  The name of a variable where the output of this operation is stored |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy in case of connectivity errors |  |
+** NONE |  Determines the level of detail about provisioned throughput consumption that is returned in the response. |  NONE | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
```

Comment:
An expression to evaluate against the operation's output. The evaluation outcome is stored in the target variable.